cmd/login_cloud.go

```diff
@@ -33,11 +33,14 @@ func getCmdLoginCloud(gs *state.GlobalState) *cobra.Command {
 	loginCloudCommand := &cobra.Command{
 		Use:   "cloud",
 		Short: "Authenticate with k6 Cloud",
-		Long: `Authenticate with k6 Cloud",
+		Long: `Authenticate with Grafana Cloud k6.
 
 This will set the default token used when just "k6 run -o cloud" is passed.`,
 		Example: exampleText,
 		Args:    cobra.NoArgs,
+		Deprecated: `This command is deprecated and will be removed in a future release.
+Please use the "k6 cloud login" command instead.
+`,
```

Comment:
Same reasoning as above:

```suggestion
		Deprecated: `and will be removed in a future release.
Please use the "k6 cloud login" command instead.
`,
```